docs/courseadministration/addcontent/learningunits/quiz_import.md

```diff
@@ -29,7 +30,7 @@ Let's have an overview of the required fields:
 
 - *Name*: the name of the quiz
 - *CourseSection*: the section of the course where the quiz is. It must be a numeric value.
-- *Quiz Type*: choose the most appropriate type for the quiz: self-test (ungraded), a weekly assignment (graded), a survey, a bonus exercise or the final exam
+- *Quiz Type*: choose the most appropriate type for the quiz: self-test (ungraded), a weekly assignment (graded, 60 min default time limit), a survey, a bonus exercise or the final exam (graded, 120 min default time limit)
```

Comment:
Hier sollte vielleicht noch angemerkt werden, dass die Zeitangaben auf der Plattform in Sekunden sind.